### PR TITLE
Add files to deploy

### PR DESCRIPTION
**Pull request by VitorDaynno**
- [Deploy](https://trello.com/c/ttDuBlGE/27-efetuar-deploy-do-servidor)

### CODE REVIEW

**Review by cesarmacedo — APPROVED** [review]

**Comment by VitorDaynno** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Full-Moon-Castle/werewolf/pullRequest?prid=5345753&bid=17501918)

File: start.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+sleep 10
```
Comment (VitorDaynno) [inline start.sh:1]: Codacy Issue found: [Tips depend on target shell and yours is unknown. Add a shebang.](https://app.codacy.com/gh/Full-Moon-Castle/werewolf/pullRequest?prid=5345753)

**Comment by VitorDaynno** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/VitorDaynno/werewolf/pullRequest?prid=5345754&bid=17501919)

File: start.sh (hunk 1 above)
Comment (VitorDaynno) [inline start.sh:1]: Codacy Issue found: [Tips depend on target shell and yours is unknown. Add a shebang.](https://app.codacy.com/manual/VitorDaynno/werewolf/pullRequest?prid=5345754)

**Pull request merged**